### PR TITLE
Update quick-start.md

### PR DESCRIPTION
**Pull request by kayleneciman**

### CODE REVIEW

**Comment by peaceiris** [comment]
This change seems unnecessary.

File: content/en/getting-started/quick-start.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ It is simple. Just call:
 hugo
 ```
 
-Output will be in `./public/` directory by default (`-d`/`--destination` flag to change it, or set `publishdir` in the config file).
+Output will be in `./private/` directory by default (`-d`/`--destination` flag to change it, or set `publishdir` in the config file).
```
Comment (peaceiris) [inline content/en/getting-started/quick-start.md]: Why `private`?

Comment (bep) [reply content/en/getting-started/quick-start.md]: I suspect this is another bogus Hacktober "free t-shirt" PR.

Comment (peaceiris) [reply content/en/getting-started/quick-start.md]: I see.

**Pull request closed without merging**